src/cron/updateTagRecommendations.ts

```diff
@@ -0,0 +1,20 @@
+import { Cron } from './cron';
+import { TagRecommendation } from '../entity/TagRecommendation';
+
+const cron: Cron = {
+  name: 'update-tag-recommendations',
+  handler: async (con, logger) => {
+    const materializedViewName =
+      con.getRepository(TagRecommendation).metadata.tableName;
+
+    try {
+      await con.query(`REFRESH MATERIALIZED VIEW ${materializedViewName}`);
```

Comment:
Materialized view is cached in db until refreshed. This allows us to have fast query for recommendations and we refresh it once a week with new keywords.

Reply:
Amazing amazing! Let's not guard it with try catch though. I'd like to employ the retry mechanism

Reply:
Got it, but if we can clear up something. Without dead letter it would retry forever? I am curious how we handle those cases in current production or do they just not happen? 

From my point of view every subscription should have dead letter just for sake of limiting number of retries.

Reply:
The only case it can happen is when you have a third-party service that you aren't sure of its availability.
Delay is high enough to deal with any production situation basically

Reply:
Granted above worker is low impact, I'll remove the try catch, just wanted to explain my logic of why I added it.